core/src/test/resources/test-crypto-key

```diff
@@ -0,0 +1 @@
+sixteenbytekey4u
```

Comment:
This new file results in a rat failure. It also need not be checked in. It can be created dynamically as part of the test, in the target directory.

Reply:
I have WriteAheadLogEncryptedIT doing this and wasn't sure which was better.  Since the file is so small its probably better to create it every time.